src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -56,11 +56,9 @@ const SubsidiesSummary = ({
 
   const hasAssignedSubsidyOrRequests = hasActiveLicenseOrLicenseRequest || hasAssignedCodesOrCodeRequests;
 
-  // We will not allow enterprises to use enterprise offers together with license/coupon codes for now
+  const hasAssignedSubsidyAndOffer = hasActiveLicenseOrLicenseRequest && canEnrollWithEnterpriseOffers;
```

Comment:
I would love a nice big block of docs somewhere in this file I think explaining what is about to happen in the display of the price. Maybe even a markdown table with the various states defined. 

It would also be fine for that to be in a PR following this one since we would like to merge sooner than later.

Reply:
https://docs.google.com/spreadsheets/d/1bdtw2zD3FVQZp6c1ah0wMkiZwGsK8McrqfqRHiG1_zo/edit?usp=sharing

WIP of logic for rendering components